server/index.js

```diff
@@ -37,6 +37,32 @@ app.get('/api/products', (req, res, next) => {
     .catch(err => next(err));
 });
 
+app.get('/api/products/:productId', (req, res, next) => {
+
+  const { productId } = req.params;
```

Comment:
Be sure to validate this value as a positive integer before going forward with the database query.